test.js

```diff
@@ -101,20 +115,47 @@ test('toVFile.read', function (t) {
     });
   });
 
-  t.test('should pass an error on non-existing files', function (st) {
-    st.plan(2);
+  t.test('should work in promise mode (string with encoding)', function (st) {
+    st.plan(3);
+
+    vfile.read('readme.md', 'utf8')
+      .then(function (file) {
+        st.equal(file.path, 'readme.md');
+        st.equal(typeof file.contents, 'string');
+        st.equal(file.toString(), fixture);
+      })
+      .catch(function () {
+        st.fail('should resolve, not reject');
```

Comment:
Hey this is an `st`!

Reply:
Well spotted! But I guess I should acutally change `t` to `st` everywhere else!